docs/jobs_reference.rst

```diff
@@ -236,6 +236,10 @@ JobDescription
 
       A name of user who created a job, :class:`str`.
 
+   .. attribute:: cluster_name
+
+      A name of cluster where job was scheduled, :class:`str`.
+
```

Comment:
Since we have CalVer and there is no scheduling strategy for the next release let's just put the current date. It is greater than the last published release and lower than the first actual release number that will include the feature.